about.html

```diff
@@ -0,0 +1,126 @@
+            </div>
+        <div><h2 class="cocogoose-rd about-main-section-item2 clip">Dont Miss any of Our Blogs</h2>
+        </div>
+        <div class="about-main-section-item3"><p>If you need help brainstorming, use a worksheet, online tools, or Google Docs to keep track of possible ideas and topics. Do word association or mind map exercises. Talk to friends and colleagues
+             (or folks on the interwebs) for ideas, or browse blogs in your niche to get a sense for what’s already been done. Use prompts. Dump all of your ideas into a single place and always carry around a notebook (or keep a virtual one, like a SwipeFile).</p>
+        </div>
+        <div class="about-main-section-item4 contact">
+            <p class="about-join"> If you want to be Blogger on our Site or if you have questions about our Blog app contact us through our E-mail</p>
+            <p class="about-email">cesblogs@gmail.com</p>
+        </div>
+    </div>
+            </section>
+            <section class="about-us">
+<h2 class="about-title">About our Blog web site</h2>
+<p class="about-discription">This Blog site was found back in 2018 and it was the best blog choise for peoples from then and we have been delivering different topics realted to
+    technology, health tips, fasion , food and more. our logo is  </p>
+    <div class="logo-box">
+    <img src="./assets/images/logo.png" alt="logo" class="about-page-logo" >
+</div>
+<hr>
+<h2 class="about-title1">Our Bloggers and the best blogg of the year</h2>
+<p class="pic-sub-title">Look our best bloggers and the best blog</p>
+<div class="our-blogger-and-top-blog">
+<div class="container-staff">
+<img src="./assets/images/blogers/staff.jpg" class="our-staff-pic" alt="our bloggers" >
+<h3 class="on-image">Our Bloggers</h3>
+<p class="image-description">our best Blogger</p>
+</div>
+<div class="container-staff">
+    <img src="./assets/images/best-paragraph.png" class="our-staff-pic" alt="our bloggers" >
+    <h3 class="on-image">Blog of the Year</h3>
+    <p class="image-description1">Written by Julia Jonson</p>
+    </div>
+</div>
+            </section>
+
+            <div class="partners dektop-hidden">
+                <h3 class="partners-title">Partner</h3>
+                <ul class="parnter-group1">
+                <li><img src="./assets/images/mozilla.svg" class="parnter1" alt="mozilla"></li>
+                <li><img src="./assets/images/google.svg" class="parnter1"  alt="google"></li>
+                <li> <h2 class="parnter2">NAVER</h2></li>
+                <li> <h2 class="parnter2 mobile-hidden">daumkakao</h2></li>
+                <li> <h2 class="parnter2 fa-brands fa-airbnb mobile-hidden">Samsung</h2></li>
+                </ul>
+                <ul class="parnter-group2">
+                    <li> <h2 class="parnter3 dektop-hidden">daumkakao</h2></li>
+                    <li> <h2 class="parnter3 dektop-hidden">Samsung</h2></li>
+                </ul>
+                </div>
```

Comment:
- [ ] Please make sure the partner's section is in the center and follow the Behance design

### Your design

![image](https://user-images.githubusercontent.com/83803414/157940293-b47cdb73-8fdd-4a2e-9d24-d2ef748552ce.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/157940483-02eb2641-ddbf-4d87-82d4-87ee6c0d2b0c.png)